cirq/contrib/paulistring/pauli_string_phasor.py

```diff
@@ -93,9 +92,6 @@ def __pow__(self, power: Union[float, value.Symbol]) -> 'PauliStringPhasor':
             return NotImplemented
         return self.extrapolate_effect(power)
 
-    def inverse(self) -> 'PauliStringPhasor':
-        return self.extrapolate_effect(-1)
-
```

Comment:
Presumably, this deletion is fine since this class already supports _pow_.  Is that correct?

Reply:
Correct.